src/shared/logs-show.c

```diff
@@ -485,6 +488,33 @@ static int output_short(
                 fprintf(f, ": [%s blob data]\n", format_bytes(bytes, sizeof(bytes), message_len));
         } else {
                 fputs(": ", f);
+
+                /* URLify config_file string in message, if the message starts with it.
+                 * Skip URLification if the highlighted pattern overlaps. */
+                if (config_file &&
+                    message_len >= config_file_len &&
+                    memcmp(message, config_file, config_file_len) == 0 &&
+                    (!highlight || highlight_shifted[0] > config_file_len)) {
```

Comment:
this doesn't work for me. i.e. this needs some `(high_shifted[0] == 0 && highlight_sifted[1] == 0)` check or so?

I am invoking "journalctl -e --no-pager" with some test data in the journal, i.e. super basic.